contracts/strategies/morpho/MetaMorphoStrategy.sol

```diff
@@ -22,6 +34,25 @@ contract MetaMorphoStrategy is ERC4626Strategy {
   /// @notice Metamorpho snippets contract, used for apr
   address public mmSnippets;
```

Comment:
It seems you forgot to remove the `morphoTransferable` storage variable just above

Reply:
we are still using it in other places in the contract actually eg in `_claimReward` and in `transferRewards`

Reply:
but at least we are not using it anymore in the `getRewardsApr` which was not needed

Reply:
Alright, feel free to close this discussion!